### PR TITLE
Fix naming in chain_properties_19 #295

### PR DESCRIPTION
**Pull request by maslenitsa93**
# Checklists

### Testing chain_properties_19

Use testnet.

1. Execute following HTML in browser:

```
<script src="node_modules/golos-js/dist/golos.min.js"></script>
Hello world
<script>
golos.config.set('websocket', 'ws://127.0.0.1:8091');
golos.api.getChainProperties(function (err, result) {
document.body.innerHTML += '<pre>' + JSON.stringify(result, null, 2) + "</pre>";
});
</script>
```

Response:
```
  ...
  "max_referral_interest_rate": 1000,
  "max_referral_term_sec": 15552000,
  "referral_break_fee": "0.000 GOLOS"
}
```

2.  Execute following cli_wallet script: 
```
./cli_wallet --server-rpc-endpoint=ws://127.0.0.1:8091
set_password qwer
unlock qwer
import_key 5JVFFWRLwz6JoP9kguuRFfytToGU6cLgBVTL9t6NB3D3BQLbUBS
info
```

Response:
```
  ...
  "max_referral_interest_rate": 1000,
  "max_referral_term_sec": 15552000,
  "referral_break_fee": "0.000 GOLOS"
}
```

3. Next, execute following in cli_wallet: 
```
update_chain_properties "cyberfounder" {"account_creation_fee": "1.000 GOLOS", "create_account_min_golos_fee": "1.000 GOLOS", "create_account_min_delegation": "1.000 GOLOS",  "create_account_delegation_time": 86400, "min_delegation": "1.000 GOLOS", "max_referral_interest_rate": 500, "max_referral_term_sec": 10552000, "referral_break_fee": "20.000 GOLOS"} true
```
Result: fail, because referral_break_fee should be between 1 and 10

Try this:
```
update_chain_properties "cyberfounder" {"account_creation_fee": "1.000 GOLOS", "create_account_min_golos_fee": "1.000 GOLOS", "create_account_min_delegation": "1.000 GOLOS",  "create_account_delegation_time": 86400, "min_delegation": "1.000 GOLOS", "max_referral_interest_rate": 500, "max_referral_term_sec": 10552000, "referral_break_fee": "0.000 GOLOS"} true
```
Result: same fail again.

Finayll, try this:
```
update_chain_properties "cyberfounder" {"account_creation_fee": "1.000 GOLOS", "create_account_min_golos_fee": "1.000 GOLOS", "create_account_min_delegation": "1.000 GOLOS",  "create_account_delegation_time": 86400, "min_delegation": "1.000 GOLOS", "max_referral_interest_rate": 500, "max_referral_term_sec": 10552000, "referral_break_fee": "10.000 GOLOS"} true
```

Result: it runs successful. Refresh webpage in browser. Response:
```
  ...
  "max_referral_interest_rate": 500,
  "max_referral_term_sec": 10552000,
  "referral_break_multiplier": "10.000 GOLOS"
}
```

### CODE REVIEW

**Comment by maslenitsa93** [comment]
Autotests passed.

File: libraries/protocol/include/golos/protocol/config.hpp [hunk 1]
```diff
@@ -114,7 +114,7 @@
 
 #define GOLOS_DEFAULT_MAX_REFERRAL_INTEREST_RATE (10*STEEMIT_1_PERCENT) // 10%
 #define GOLOS_DEFAULT_MAX_REFERRAL_TERM_SEC     (60*60*24*30*6)
-#define GOLOS_DEFAULT_REFERRAL_BREAK_FEE        (10*STEEMIT_MIN_ACCOUNT_CREATION_FEE)
+#define GOLOS_DEFAULT_MAX_REFERRAL_BREAK_FEE    (10*STEEMIT_MIN_ACCOUNT_CREATION_FEE)
```
Comment (afalaleev) [inline libraries/protocol/include/golos/protocol/config.hpp]: Replace this with `asset(100, STEEM_SYMBOL)`

File: libraries/protocol/include/golos/protocol/config.hpp [hunk 2]
```diff
@@ -327,7 +327,7 @@
...
```
Comment (afalaleev) [inline libraries/protocol/include/golos/protocol/config.hpp]: Replace this with `asset(100, STEEM_SYMBOL)`

File: libraries/protocol/include/golos/protocol/steem_operations.hpp [hunk 3]
```diff
@@ -515,8 +515,8 @@ namespace golos { namespace protocol {
             /**
              * Fee for breaking referral deductions by referral
              */
-            asset referral_break_fee =
-                    asset(GOLOS_DEFAULT_REFERRAL_BREAK_FEE, STEEM_SYMBOL);
+            asset max_referral_break_fee =
+                    asset(GOLOS_DEFAULT_MAX_REFERRAL_BREAK_FEE, STEEM_SYMBOL);
```
Comment (afalaleev) [inline libraries/protocol/include/golos/protocol/steem_operations.hpp]: I described GOLOS_DEFAULT_MAX_REFERRAL_BREAK_FEE as asset, so there should be GOLOS_DEFAULT_MAX_REFERRAL_BREAK_FEE without asset

File: libraries/protocol/steem_operations.cpp [hunk 4]
```diff
@@ -230,7 +230,7 @@ namespace golos { namespace protocol {
             chain_properties_18::validate();
             GOLOS_CHECK_VALUE_LE(max_referral_interest_rate, STEEMIT_100_PERCENT);
             GOLOS_CHECK_VALUE_LE(max_referral_term_sec, 60*60*24*30*12);
```
Comment (afalaleev) [inline libraries/protocol/steem_operations.cpp]: 60*60*24*30*12 - > to constants in config.hpp

**Comment by maslenitsa93** [comment]
Do not write excessive (unpredictable stylistic) comments, please. Each issue has deadline. Issue completed - merged - comments wrote.

**Comment by afalaleev** [comment]
At the next commit add protection for HF - `STEEMIT_HARDFORK_0_19__295`

**Pull request merged**